charts/registry-scanner/templates/secret.yaml

```diff
@@ -12,6 +12,9 @@ data:
   aws_access_key_id: {{ .Values.config.aws.accessKeyId | b64enc | quote }}
   aws_secret_access_key: {{ .Values.config.aws.secretAccessKey | b64enc | quote }}
   aws_region: {{ required "A valid .Values.config.aws.region is required" .Values.config.aws.region | b64enc | quote }}
```

Comment:
What's happened with the "gcr" type check here? I still see the code in the comment https://github.com/sysdiglabs/charts/pull/1354#discussion_r1340287484 but not in the final version. I think we should keep it, as we list gcr in the available registry types

Reply:
For the moment, we have GCR on docs PR as not officially supported. We have 2 options:
- Keep the code as we have previously just o give the possibility to show GCR well on UI (but not officially supported)
- Keep the code as on the final version without a mention to GCR. User can use it as previous to this FR but at least for me, no makes sense to have it when google cloud not give us the possibility to create a new GCR on our accounts because is deprecated and will be removed on google cloud on a few months.

What do you prefer @airadier ?

Reply:
@jprieto92 I would keep any code that helps supporting GCR (so the branch in the template that automatically populates the user helps), but no mention either in the docs or in the values.yaml. We might add it in a few days, as we are getting some requests from customers, so it would be just a matter of updating the documentation, but let's keep the code and template as "ready for GCP" as possible. Deal?

Reply:
after take a look together @airadier latest change here ready to review

Reply:
done, resolving conversation @airadier